### ISSUE TITLE
Uses depreciated library

### ISSUE THREAD

**Issue by JRSHENK** [RobotDynOfficial/RBDDimmer#76]
esp_intr.h is used in RBDDimmer and has been depreciated and replaced with esp_intr_alloc.h. This causes the inability to compile using Arduino IDE if ESP by Espresive is updated to V3.0.0.

**Comment by Arulamir** [comment]
> esp_intr.h is used in RBDDimmer and has been depreciated and replaced with esp_intr_alloc.h. This causes the inability to compile using Arduino IDE if ESP by Espresive is updated to V3.0.0.

yes it can be solution, but after that the probrlem changed in file .cpp
